Autopilot/Test/Src/TelemetryManager/Test_TelemetryFSM.cpp

```diff
@@ -0,0 +1,110 @@
+#include <gtest/gtest.h>
+
+#include "telemetryManager.hpp"
+#include "telemetryStateClasses.hpp"
+
+
+using namespace std; 
+using ::testing::Test;
+
+TEST(TelemetryManagerFSM, InitialStateIsInitialMode){
+    /***SETUP***/
+    telemetryManager telemetryMng;
+    /***ASSERTS***/
+    ASSERT_EQ(*(telemetryMng.getCurrentState()), initialMode::getInstance());
+}
+
+TEST(TelemetryManagerFSM, InitialTransitionToGround){
+    /***SETUP***/
+    telemetryManager telemetryMng;
+    /***STEPTHROUGH***/
+    telemetryMng.execute();
```

Comment:
I think you have to set the state of the telemetryMng before executing the state. 

If you look at the AttitudeManager FSM tests you'll see what I mean